packages/core/lib/debug/cli.js

```diff
@@ -133,13 +133,15 @@ class CLIDebugger {
     const startMessage = DebugUtils.formatStartMessage(
       this.txHash !== undefined
     );
+    const registry = this.config.noEns ? null : this.config.registry;
```

Comment:
We have ENS registry configuration options elsewhere in truffle-config.js. Is that scoped specifically to the deployer? I'm not opposed to merging this with the duplication, but could you please at least open an issue to note this as something we'll want to resolve?

@eggplantzzz might be the person to ask about this... it doesn't look like it's in our configuration docs.

Reply:
(Oh, that is to say... keeping `--registry` is fine as a CLI option, but if the user doesn't specify this, then `truffle debug` should first look to see if it can find the setting in the config)

Reply:
Oh, huh -- yeah I hadn't considered that.  Good idea to check that, will have to take a look at that.

Reply:
Oh dear, on taking a look at this, I ended up having to write up #5997!

Reply:
OK, I've merged in #6005 and updated this PR appropriately!

Reply:
Awesome!